static/src/javascripts/projects/commercial/modules/prebid/utils.js

```diff
@@ -119,10 +119,13 @@ export const shouldIncludeAdYouLike = (slotSizes: PrebidSize[]): boolean => {
 export const shouldIncludeOzone = (): boolean =>
     !isInUsRegion() && !isInAuRegion();
 
-export const shouldIncludeAppNexus = (): boolean =>
-    isInAuRegion() ||
-    ((config.get('switches.prebidAppnexusUkRow') && !isInUsRegion()) ||
-        !!pbTestNameMap().and);
+export const shouldIncludeAppNexusAu = (): boolean => isInAuRegion();
```

Comment:
minor: is this check likely to get more complicated, and do we benefit from the alias here?

We could potentially nix `shouldIncludeAppNexusAu`, since I think we only use it once? and directly query `isInAuRegion` :

```js
        ...(inPbTestOr(isInAuRegion()) ? [appNexusBidderAu] : []),
```

Reply:
No, it won't.